### PR TITLE
CERT-3448 Fix the documentation for gambit a bit

### PR DESCRIPTION
**Pull request by chandrakananandi**
Links:
 - [Gambit](https://certora-certora-prover-documentation--150.com.readthedocs.build/en/150/docs/gambit/gambit.html)
 - [Mutation verifier](https://certora-certora-prover-documentation--150.com.readthedocs.build/en/150/docs/gambit/mutation-verifier.html)

### CODE REVIEW

File: docs/gambit/gambit.md [hunk 1]
```diff
@@ -242,12 +242,11 @@ Here are some examples of using the `--sourceroot` option.
    ```
 
    This should output the following:
-   <!-- Code output: using `pre` to avoid the Copy To Clipboard feature -->
-   <pre>
+   ```
```
Comment (mdgeorge4153) [inline docs/gambit/gambit.md:245]: I'm fine with this change, although IIRC this code is automatically generated; is this change in sync with changes in the original source?

File: docs/gambit/mutation-verifier.md [hunk 2]
```diff
@@ -42,10 +42,9 @@ certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/m
 ```{note}
 You must run `certoraMutate` from the root of the Solidity project directory.
 The files `prover.conf` and `mutation.conf`
-can be in their own directories.
+can be in their own directories but always within the project directory.
```
Comment (mdgeorge4153) [inline docs/gambit/mutation-verifier.md:45]:
```suggestion
can be in their own directories, but must always be within the project directory.
```

**Review by mdgeorge4153 — APPROVED** [review]
Looks good, as long as it is in sync with the auto-generation

File: docs/gambit/gambit.md (hunk 1 above)
Comment (chandrakananandi) [reply docs/gambit/gambit.md:245]: You are right that we have a PR for auto generated doc, but that PR is on hold for a bit. When that PR lands, it is expected to handle this correctly (i.e., change it to ``` when needed). Here, this was just a copy paste error.

**Pull request merged**